ingest/src/main/scala/hydra.ingest/services/IngestionFlowV2.scala

```diff
@@ -96,4 +99,34 @@ object IngestionFlowV2 {
   final case class AvroConversionAugmentedException(message: String) extends RuntimeException(message)
   final case class SchemaNotFoundAugmentedException(schemaNotFoundException: SchemaNotFoundException, topic: String)
     extends RuntimeException(s"Schema '$topic' cannot be loaded. Cause: ${schemaNotFoundException.getClass.getName}: Schema not found for $topic")
+
+  final case class KeyAndValueMismatch(fieldName: String, keyValue: AnyRef, valValue: AnyRef)
+  final case class KeyAndValueMismatchedValuesException(mismatches: List[KeyAndValueMismatch])
+    extends RuntimeException(
+      (List(s"Fields that exist in key schema and value schema must have same value.","Field Name\tKey Value\tValue Value") ++
+        mismatches.map(m => s"${m.fieldName}\t${m.keyValue.toString}\t${m.valValue.toString}")).mkString("\n")
```

Comment:
Any issues with this format if the fields are larger than the tabs? I like the format, just worried about the columns matching for larger fields.

Reply:
They don't line up perfectly anyway. I think the separation, though imperfect, is still more helpful than not having it at all. If they put it into a better editor (some spreadsheet) it will line up quite nicely. Postman, for example, doesn't honor tabs, it just adds 4 spaces from the previous character which ruins alignment.